### DISCUSSION TITLE
Change default date range

### DISCUSSION THREAD

**Discussion by VictorDP90** [firefly-iii/firefly-iii#8111] [General]
Hello.

I'd love to have the possibility of set default a custom date range, I know I can set it but when I log off and log in, I have to set it again. There is any option to set it by default?

I always get my salary on 25th and I would like to see my balance from 25th to 25th of the next month.

Thanks!

**Comment by JC5** [comment]
This isn't possible I'm afraid.

https://docs.firefly-iii.org/firefly-iii/more-information/what-its-not/
